GETTING STARTED - Developers.rdoc

```diff
@@ -0,0 +1,61 @@
+==Is this doc for you?
+
+This page offers a Developer's introduction to the Wagn code base and how to customize it for your own needs. it assumes you're a ruby developer or, at least, someone who does not fear ruby code.
+
+It also assumes that you've experimented with Wagn enough to have some familiarity with the basics (cards, types, rules, etc.) <b>If you have no idea what a card is, you may find it helpful to begin at {http://wagn.org Wagn.org}</b> to get a sense of what Wagn's all about.
+
+And, finally, it assumes you're looking at it from a docs site (ie, NOT GitHub).  The page is peppered with links; if they're breaking, try FIXME - (give links here).
+
```

Comment:
First, I would like to offer my start at a multi-level description of the architecture and more: http://gerry.wagn.org/It_s_Cards_all_the_Way_Down  To the extent that it is wrong or incomplete, please comment and contribute.  If you'd rather move it to the wagn.org space, that's fine with me too.

In general, I always understood we want to document stuff in wagn, pretty much regardless of target audience.  I realize there is some tension with code documentation and Yard.  We may need some glue on either side (yard or wagn/decko).